app/models/study.rb

```diff
@@ -763,6 +763,8 @@ def has_bucket_access?(user)
     else
       if self.user == user
         return true
+      elsif user.admin? # need a separate case for this since admin users won't be in user_in_group_share?
+        return true
```

Comment:
The reason that this method had no special case for admins is that "admin" accounts only have special access in the portal, not in Terra or the bucket.  So we wouldn't want to return true here - it will almost never be the case.